spec/factories/passing_game_log.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :passing_game_log do
+    sequence(:completions, (20..30).cycle) { |n| n }
+    sequence(:passing_attempts, (40..60).cycle) { |n| n }
+    sequence(:passing_yards, (200..400).cycle) { |n| n }
+  end
```

Comment:
Great use of `cycle`, I've never seen this before in a factory. I'll have to remember this one, thanks!